### PR TITLE
initial PR for genAI

### PR DESCRIPTION
**Pull request by atbida**

### CODE REVIEW

**Comment by atbida** [comment]
Note to self: contact@prophecy.io needs to be updated.

**Comment by slyons** [comment]
There's a missing image as well

File: docs/getting-started/genaichatbot.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+![Chatbot Live Pipeline](img/genai_chatbot_live.png)
+
+#### 3a.1 Configuring the Chatbot Live Pipeline
+
+1. Verify the `slack_chat` Source Gem is configured with Databricks scope `slack` and Databricks Key `app_token`. While this token begins with `xapp-`, be sure not to use the plaintext value, as using the Databricks secret is a much more secure approach.
+2. Update the `only_user_msgs` Filter Gem with the Slack app member ID:
+   ![Slack App Member ID](img/genai_memberId.png)
```
Comment (atbida) [inline docs/getting-started/genaichatbot.md:178]: @slyons thanks for pointing out this image was missing! 
I added it. Did you notice another image is missing?

Comment (slyons) [reply docs/getting-started/genaichatbot.md:178]: That was the only one

File: docs/getting-started/getting-started-with-low-code-sql.md [hunk 2]
```diff
@@ -389,6 +389,6 @@ You've successfully set up, developed, tested, and deployed your first SQL proje
 
 To continue learning and expanding your skills with Prophecy, explore other tutorials within our documentation, or apply your newfound knowledge to address real-world business challenges!
 
-If you ever encounter any difficulties, don't hesitate to reach out to us at Support@Prophecy.io or join our [Slack community](https://prophecy-io-support.slack.com/join/shared_invite/zt-moq3xzoj-~5MSJ6WPnZfz7bwsqWi8tQ#/shared-invite/email) for assistance. We're here to help!
+If you ever encounter any difficulties, don't hesitate to Contact.us@Prophecy.io or join our [Slack community](https://prophecy-io-support.slack.com/join/shared_invite/zt-moq3xzoj-~5MSJ6WPnZfz7bwsqWi8tQ#/shared-invite/email) for assistance. We're here to help!
```
Comment (slyons) [inline docs/getting-started/getting-started-with-low-code-sql.md:392]: I still think you need "don't hesitate to reach out to us at" in front of the email, unless we're being clever using the email address, in which case disregard

**Review by slyons — APPROVED** [review]

**Review by several27 — APPROVED** [review]

**Pull request merged**